package.json

```diff
@@ -34,10 +34,21 @@
   "homepage": "https://github.com/remcohaszing/monaco-tailwindcss#readme",
   "dependencies": {
     "@ctrl/tinycolor": "^3.0.0",
+    "color-name": "^1.0.0",
+    "css.escape": "^1.0.0",
+    "line-column": "^1.0.0",
     "monaco-marker-data-provider": "^1.0.0",
     "monaco-worker-manager": "^1.0.0",
+    "moo": "^0.5.0",
+    "multi-regexp2": "^1.0.0",
+    "postcss": "^7.0.0",
```

Comment:
i remebered postcss not being webcompatible -> eg we need to browserfy stuff?
but thats where this postcss-js stuff comes into play?

Reply:
I used the demo example to make sure it still works without modifications. The esbuild example can be built (CI tests this), but I didn’t try running it.

My goal is to be able to consume this package without having to deal with special dependency setups. Since the demo works, I think it’s fine?

Reply:
yes this is my goal too ^^ so i was just making sure that this would still be possible. btw i think the esbuild demo is not build right? https://github.com/remcohaszing/monaco-tailwindcss/runs/5960979783?check_suite_focus=true because i missed making a build npm script ...

Reply:
I right! I didn’t notice this before. I’m surprised npm doesn’t fail on a missing script.

Reply:
postcss-js is not new it was previously also installed and required by tailwindcss. I see. And its not doing what i assumed (i thought at first it was some browserfied post css)

Reply:
postcss should be then removed from dev deps i guess
https://github.com/remcohaszing/monaco-tailwindcss/pull/25/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R65